bundles/org.openhab.binding.rfxcom/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -30,6 +30,13 @@
 		<state min="0" max="255" step="1" pattern="%d" readOnly="false"></state>
 	</channel-type>
 
+	<channel-type id="commandString">
+		<item-type>String</item-type>
+		<label>Command String</label>
+		<description>Command channel, Name of the channel</description>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="false"></state>
```

Comment:
Channel definition of `commandString` channel should probably not have a state with decimal types.